src/core/Akka.Streams/Implementation/IO/FilePublisher.cs

```diff
@@ -101,6 +108,9 @@ protected override void PreStart()
             try
             {
                 _chan = _f.Open(FileMode.Open, FileAccess.Read);
+                if (_startPosition > 0) {
```

Comment:
either no curly braces or in the next line, but no java style :stuck_out_tongue: